mm/iob/Kconfig

```diff
@@ -41,7 +41,7 @@ config IOB_HEADSIZE
 
 config IOB_ALIGNMENT
 	int "Alignment size of each I/O buffer"
-	default 1
+	default 8
```

Comment:
I think taking into account specific of a NuttX network subsystem the default alignment should be 2.

Reply:
OK